### DISCUSSION TITLE
Some docs are missing in new documentation site, should we offer a way to go back?

### DISCUSSION THREAD

**Discussion by Akarinnnnn** [MonoGame/MonoGame#8149] [General]
1 hour before I'm attempting to read docs about [Messagebox class](https://docs.monogame.net/api/Microsoft.Xna.Framework.Input.MessageBox.html)(this link is provided by search engine). But I've been redirected to the homepage of new documentation site. There seems no corresponding documentation exists on new site. Can I obtain an archive of old docs?

**Comment by Bletraut** [comment]
Maybe this link will help
[Message Box](https://monogame.net/api/Microsoft.Xna.Framework.Input.MessageBox/)

And this
[Api](https://monogame.net/api/)

**Reply by Akarinnnnn** [reply]
Thank you, this helped a lot!

**Comment by ThomasFOG** [comment]
Documentation is being migrated from ```docs.monogame.net``` to ```monogame.net```, we're working on a redirection.

**Reply by Akarinnnnn** [reply]
Thanks, and I'd like to add an example about using `MessageBox`, how can I get started?

**Reply by ThomasFOG** [reply]
If you'd like to improve the API reference with examples, you can do that directly in the code's comments, like it is made on this line: https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Input/MessageBox.cs#L20.

If you propose changes through a pull request, it will update the documentation on the website.

**Comment by AristurtleDev** [comment]
This should be resolved once PR https://github.com/MonoGame/monogame.github.io/pull/91 gets pushed through.

This happened because the switch to 11ty generated "pretty urls" where the original docfx generated "ugly urls", which caused the search engine result links to not redirect properly.
